docs/markdown/Reference-tables.md

```diff
@@ -163,6 +163,31 @@ These are provided by the `.system()` method call.
 Any string not listed above is not guaranteed to remain stable in
 future releases.
 
+## Kernel names (since 1.2.0)
+
+Native names as returned by the `.kernel()` method.
+
+| Value               | Comment                         |
+| -----               | -------                         |
+| linux   | |
+| freebsd | |
+| openbsd | |
+| windows | |
```

Comment:
Should we maybe call this `nt`? To differentiate from the `ce` kernel, for instance or any future windows kernels. Also less surprising if `nt` is returned here while building on `cygwin`.

I think these are also missing: `sunos` `dragonfly` `haiku` `netbsd`

I also wonder if we should have a special value when targeting embedded / bare metal systems that do not use a kernel at all, like with AVR-GCC, or when targeting `emscripten`.

Reply:
I wonder if Solaris and illumos should return different values instead of both returning sunos for kernel, since they are different kernels?

@alanc, what do you think?

Reply:
It would make sense to me to differentiate Solaris & illumos, since they will offer somewhat differing sets of kernel interfaces & features, much like the different BSD's do, despite a common ancestor, and the difference will continue to grow over time.

Reply:
Yes, in general we're trying to get illumos identified explicitly and separately from Solaris in toolchains, so I would be in favour of `illumos` showing up here as well.  You can always CC me on illumos questions if you need anything!